app/billing_client.go

```diff
@@ -0,0 +1,310 @@
+package app
+
+import (
+	"context"
+	"errors"
+	"io"
+
+	pb "go.viam.com/api/app/v1"
+	"go.viam.com/utils/rpc"
+	"google.golang.org/protobuf/types/known/timestamppb"
+)
+
+// UsageCostType specifies the type of usage cost.
+type UsageCostType int32
+
+const (
+	// UsageCostTypeUnspecified is an unspecified usage cost type.
+	UsageCostTypeUnspecified UsageCostType = iota
+	// UsageCostTypeDataUpload represents the usage cost from data upload.
+	UsageCostTypeDataUpload
+	// UsageCostTypeDataEgress represents the usage cost from data egress.
+	UsageCostTypeDataEgress
+	// UsageCostTypeRemoteControl represents the usage cost from remote control.
+	UsageCostTypeRemoteControl
+	// UsageCostTypeStandardCompute represents the usage cost from standard compute.
+	UsageCostTypeStandardCompute
+	// UsageCostTypeCloudStorage represents the usage cost from cloud storage.
+	UsageCostTypeCloudStorage
+	// UsageCostTypeBinaryDataCloudStorage represents the usage cost from binary data cloud storage.
+	UsageCostTypeBinaryDataCloudStorage
+	// UsageCostTypeOtherCloudStorage represents the usage cost from other cloud storage.
+	UsageCostTypeOtherCloudStorage
+	// UsageCostTypePerMachine represents the usage cost per machine.
+	UsageCostTypePerMachine
+)
+
+func usageCostTypeFromProto(costType pb.UsageCostType) UsageCostType {
```

Comment:
[opt] Could hypothetically put these "from proto" methods in `app/billing_proto_conversions.go` to match your pattern in the app wrappers PR; up to you though.

Reply:
It's only in app that we're doing that. In fact, I think we might be removing that file for app as well.